orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -3866,3 +3868,34 @@ def test_ortmodule_determinism_flag(is_training,deterministic):
 
         from onnxruntime.training.ortmodule import _are_deterministic_algorithms_enabled
         assert _are_deterministic_algorithms_enabled() is torch.are_deterministic_algorithms_enabled()
+
+
+def test_ortmodule_gradient_builder():
+    class Model(torch.nn.Module):
+        def __init__(self):
+            super(Model, self).__init__()
+        def forward(self, x):
+            return torch.cos(x)
+
+    device = 'cuda'
+
+    @register_gradient('', 'Cos')
+    def Cos_gradient():
+        return [('Sin', ['GO(0)'], ['Sin_X']),
+                ('Neg', ['Sin_X'], ['GI(0)'])]
+
+    pt_model = Model().to(device)
+    debug_options = DebugOptions(save_onnx=True, onnx_prefix='cos')
```

Comment:
don't need onnx_prefix='cos', as cos should be exported a normal onnx op

Reply:
Why do we need to save onnx here? It seems unnecessary as the test is not consuming the onnx file in any moment. That can waste space on CI VMs or even cause a build error when the same test run more than once (the save API would fail because an identical file already exists)